packages/frontend/src/utils/getWalletURL.js

```diff
@@ -0,0 +1,14 @@
+import { IS_MAINNET, SHOW_PRERELEASE_WARNING } from '../config';
+
+export default (https = true) => {
+    let networkName = '';
+
+    if (SHOW_PRERELEASE_WARNING) {
+        networkName = 'staging.';
+    }
+    if (!IS_MAINNET) {
```

Comment:
Could we make this an `else if` just to make it clear we expect `SHOW_PRERELEASE_WARNING` and `!IS_MAINNET` to be mutually exclusive?